app/views/hyrax/embargoes/_list_expired_active_embargoes.html.erb

```diff
@@ -1,25 +1,22 @@
 <% if assets_with_expired_embargoes.blank? %>
-  <table  class="embargoes table">
+
+  <table class="embargoes table">
+    <%= render partial: 'table_header' %>
     <tbody>
-    <tr>
-      <th>Type of Work</th><th>Title</th><th>Current Visibility</th><th>Embargo Release Date</th><th>Visibility will Change to</th>
-    </tr>
-    <tr>
-      <td colspan="5" class="text-center"><p>There are no expired embargoes in effect at this time.</p> </td>
-    </tr>
+      <tr>
+        <td colspan="5" class="text-center">
+          <p><%= t('.missing') %></p>
+        </td>
+      </tr>
     </tbody>
   </table>
 
 <% else %>
 
   <%= form_tag embargoes_path, method: :patch do %>
-    <%= submit_tag 'Deactivate Embargoes for Selected', class: 'btn btn-primary' %>
+    <%= submit_tag t('.deactivate_embargoes_for_selected'), class: 'btn btn-primary' %>
     <table class="embargoes table">
-      <thead>
-        <tr>
-          <th></th><th>Type of Work</th><th>Title</th><th>Current Visibility</th><th>Embargo Release Date</th><th>Visibility will Change to</th><th></th>
-        </tr>
-      </thead>
+      <%= render partial: 'table_header' %>
```

Comment:
This partial substitution might not work, because the current header has an extra empty column.